packages/core/src/domain/configuration/configuration.ts

```diff
@@ -4,12 +4,12 @@ import { catchUserErrors } from '../../tools/catchUserErrors'
 import { includes, objectHasValue, ONE_KILO_BYTE, ONE_SECOND } from '../../tools/utils'
 import { computeTransportConfiguration, TransportConfiguration } from './transportConfiguration'
 
-export const InitialPrivacyLevel = {
+export const DefaultPrivacyLevel = {
   ALLOW: 'allow',
   MASK: 'mask',
-  MASK_FORMS_ONLY: 'mask-forms-only',
+  MASK_USER_INPUT: 'mask-user-input',
```

Comment:
Shouldn't it be plural 'mask-user-inputs'

Reply:
I don't really think so, as it's not talking about the `<input>` elements but rather user input as a concept of recorded data.